alephium/.project.json

```diff
@@ -203,7 +205,12 @@
       "sourceCodeHash": "81c5941101098365861356351470f40d88b863ddb1546e57218e3756c1f483bc",
       "bytecodeDebugPatch": "",
       "codeHashDebug": "3921ab78ca073b5125b2ff7e4232e72f2c1c62554bbc8f81d39e769e6bcf349c",
-      "warnings": []
+      "warnings": [
```

Comment:
Side note (not for this PR): In web3 SDK, we could remove warnings when saving the file.